src/palace/manager/feed/serializer/opds.py

```diff
@@ -79,7 +84,7 @@ def _tag(
 
     def _attr_name(self, attr_name: str, mapping: dict[str, str] | None = None) -> str:
         if not mapping:
-            mapping = ATTRIBUTE_MAPPING
+            mapping = V1_ATTRIBUTE_MAPPING
```

Comment:
Since we defined `_get_attribute_mapping`, shouldn't this be calling that function?